extensions/podman/src/extension.ts

```diff
@@ -331,7 +424,102 @@ export async function activate(extensionContext: extensionApi.ExtensionContext):
   await registrySetup.setup(extensionContext);
 }
 
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+function startPodmanUpdateCheck(): void {
+  const currentTime = new Date().getMilliseconds();
+  const oneDay = 86400000;
+  if (currentTime - podmanInfo.lastUpdateCheck < oneDay) {
+    doCheckUpdate();
+  } else {
+    // setup timer
+    updateCheckTimer = setTimeout(() => doCheckUpdate(), oneDay - currentTime - podmanInfo.lastUpdateCheck);
+  }
+}
+
+async function doCheckUpdate(): Promise<void> {
+  const latestVersion = await fetchLatestPodmanVersion();
+  if (latestVersion.name !== podmanInfo.podmanVersion) {
+    //TODO: better to use some sort of notifications for this
+    const answer = await extensionApi.window.showDialog(
+      'question',
+      'Podman Update',
+      `There are new Podman ${latestVersion.name}.\nDo you want to install it?`,
+      'Yes',
+      'No',
+    );
+    if (answer === 'Yes') {
+      await installPodman(latestVersion);
+      const newInstalledPodman = await getPodmanInstallation();
+      // write podman version
+      if (newInstalledPodman) {
+        podmanInfo.podmanVersion = newInstalledPodman.version;
+      }
+    }
+  }
+
+  podmanInfo.lastUpdateCheck = new Date().getMilliseconds();
+}
+
+async function checkInstalledPodmanVersion(installedVersion: string, bundledVersion: string): Promise<void> {
+  if (lte(installedVersion, bundledVersion)) {
+    const answer = await extensionApi.window.showDialog(
```

Comment:
we need a 'skip Version' as well as if people don't want to update to this version, this popup should not come everyday

Also we should propose this flow only if we're able to do it